api/src/controllers/put/mine.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require('mongoose');
+const ObjectId = require('mongoose').Types.ObjectId;
+
+const SYSTEM_USER = 'SYSTEM_USER';
+
+/**
+ * Performs all operations necessary to edit a master Mine record.
+ *
+ * @param {*} args
+ * @param {*} res
+ * @param {*} next
+ * @param {*} incomingObj see example
+ * @returns object containing the operation's status and created records
+ */
+exports.editRecord = async function(args, res, next, incomingObj) {
```

Comment:
In keeping with the other controllers, this should be called `editMaster` and then there should be a separate `editRecord` function that makes all the calls needed to do all the editing. I think Daniel has an open PR where he changed some of this though, so depending on merge order, might be a bit weird to sync up on.

Reply:
Updated to match the structure of the other controllers.